requirements-dev.txt

```diff
@@ -4,6 +4,7 @@ pyarrow>=4.0.1
 dask[complete]>=2.22.0,<2022.2.0
 distributed>=2.22.0,<2022.2.0
 ray[default]>=1.4.0
+redis
```

Comment:
Should we use pin for `redis` as Ray guys use: https://github.com/ray-project/ray/pull/23991 ?

Reply:
Yes, we should.

Reply:
done, made the pin for `redis>=3.5.0,<4.0.0`